### PR TITLE
change not valid to invalid

### PR DESCRIPTION
**Pull request by foxyriver**
**What this PR does / why we need it**:

change not valid to invalid

```release-note
   NONE
```

### CODE REVIEW

File: pkg/apis/apps/validation/validation.go [hunk 1]
```diff
@@ -108,7 +108,7 @@ func ValidateStatefulSetSpec(spec *apps.StatefulSetSpec, fldPath *field.Path) fi
 	} else {
 		allErrs = append(allErrs, unversionedvalidation.ValidateLabelSelector(spec.Selector, fldPath.Child("selector"))...)
 		if len(spec.Selector.MatchLabels)+len(spec.Selector.MatchExpressions) == 0 {
-			allErrs = append(allErrs, field.Invalid(fldPath.Child("selector"), spec.Selector, "empty selector is not valid for statefulset."))
+			allErrs = append(allErrs, field.Invalid(fldPath.Child("selector"), spec.Selector, "empty selector is invalid for statefulset."))
```
Comment (nikhita) [inline pkg/apis/apps/validation/validation.go]: while we are here, can you remove the period at the end? In Go, it's suggested not to use punctuation at the end of the error message.

File: pkg/apis/core/v1/helper/helpers.go [hunk 2]
```diff
@@ -79,7 +79,7 @@ func HugePageResourceName(pageSize resource.Quantity) v1.ResourceName {
 // an error is returned.
 func HugePageSizeFromResourceName(name v1.ResourceName) (resource.Quantity, error) {
 	if !IsHugePageResourceName(name) {
-		return resource.Quantity{}, fmt.Errorf("resource name: %s is not valid hugepage name", name)
+		return resource.Quantity{}, fmt.Errorf("resource name: %s is invalid hugepage name", name)
```
Comment (nikhita) [inline pkg/apis/core/v1/helper/helpers.go]: would be better worded as `is an invalid hugepage name`

File: pkg/apis/core/helper/helpers.go [hunk 3]
```diff
@@ -56,7 +56,7 @@ func HugePageResourceName(pageSize resource.Quantity) core.ResourceName {
 // an error is returned.
 func HugePageSizeFromResourceName(name core.ResourceName) (resource.Quantity, error) {
 	if !IsHugePageResourceName(name) {
-		return resource.Quantity{}, fmt.Errorf("resource name: %s is not valid hugepage name", name)
+		return resource.Quantity{}, fmt.Errorf("resource name: %s is invalid hugepage name", name)
```
Comment (nikhita) [inline pkg/apis/core/helper/helpers.go]: `is an invalid hugepage name` would be better.

**Comment by foxyriver** [comment]
@nikhita PTAL

**Comment by nikhita** [comment]
/ok-to-test
/kind cleanup
/lgtm

@foxyriver thanks for the fix!

**Comment by nikhita** [comment]
/assign @liggitt 
for approval

**Comment by nikhita** [comment]
/lgtm

@foxyriver thanks for rebasing quickly!

**Pull request merged**